library/build.gradle

```diff
@@ -6,7 +6,7 @@ version = rootProject.ext.versionName
 
 android {
     compileSdkVersion rootProject.ext.compileSdkVersion
-    buildToolsVersion rootProject.ext.buildToolsVersion
+    buildToolsVersion = "29.0.2"
```

Comment:
I don't think you need to change this, if you already changed the root `build.gradle`? Mainly because then it becomes 2 places you need to make changes in the future.